docs/user/openstack/install_upi.md

```diff
@@ -345,13 +345,22 @@ api_vips = [inventory["os_apiVIP"]]
 ingress_vips = [inventory["os_ingressVIP"]]
 ctrl_plane_port = {"network": {"name": inventory["os_network"]}, "fixedIPs": [{"subnet": {"name": inventory["os_subnet"]}}]}
 if inventory.get("os_subnet6"):
-    machine_net.append({"cidr": inventory["os_subnet6_range"]})
-    api_vips.append(inventory["os_apiVIP6"])
-    ingress_vips.append(inventory["os_ingressVIP6"])
-    data["networking"]["networkType"] = "OVNKubernetes"
-    data["networking"]["clusterNetwork"].append({"cidr": inventory["cluster_network6_cidr"], "hostPrefix": inventory["cluster_network6_prefix"]})
-    data["networking"]["serviceNetwork"].append(inventory["service_subnet6_range"])
-    ctrl_plane_port["fixedIPs"].append({"subnet": {"name": inventory["os_subnet6"]}})
+    if inventory.get("ipv6_primary"):
```

Comment:
To simplify things, we could remain handling the `inventory.get("os_subnet6")` condition how we used before and inside of it add this condition `if inventory.get("ipv6_primary")` that would just do a `.reverse()` in the necessary lists.

Reply:
That's a good idea.